Unpacked/Outgoing_v3/Outgoing_v3.js

```diff
@@ -1,277 +1,271 @@
 		var array = string.split(delimeter);
 		return array;
 	}
-
-   jobTicketPath = jobTicketLocation + getDirectorySeperator(s) + 'ticket.xml';
-
-	var doc = new Document(jobTicketPath);
-	if(!doc.isWellFormed()){
+	jobTicketPath = jobTicketLocation + getDirectorySeperator(s) + 'ticket.xml';
+	var jobTicketFile = new File(jobTicketPath);
+	var jobTicketFileUsable = isFileUsable(s, jobTicketFile);	
+	if(jobTicketFileUsable === false) {
 		// Something is wrong, read the doc for debugging
-		malformedTicket = File.read(jobTicketPath, "UTF-8");
-		job.log(3, "Malformed ticket: "+ malformedTicket);
+		job.log(3, "ticket does not exist");
 		// Fail the dummy job
 		job.fail("Job ticket could not be restored and the job could not be processed.");
-		fatalJobCleaner(s, job, filePath); // Kill the failure
-	}
-	var docChildren = doc.getChildNodes();
-	var jobTicketNode = docChildren.getItem(0);
-
-	var children = jobTicketNode.getChildNodes();
-
-	var i, node, key, value, textNode, oldUniqueNamePrefix, oldName;
-
-	// Loop through job ticket nodes
-	for(i = 0; i < children.getCount();i++){
-
-		node = children.getItem(i);
-
-		key = node.getBaseName();
-
-		if(key !== 'getPrivateData'){
-			// Standard job ticket field
-			textNode = node.getFirstChild();
-			if(textNode){
-				value = textNode.getValue();
-
-				// Log
-				if(verboseDebugging === true){
-					s.log(-1, 'Restoring job ticket part - ' + key + ': ' + value);
-				}
-
-				// Restore
-				if(key == 'getJobState') job.setJobState(value);
-				if(key == 'getPriority') job.setPriority(value);
-				if(key == 'getEmailBody') job.setEmailBody(value);
-				if(key == 'getEmailAddresses') job.setEmailAddresses(splitString(value, ","));
-				if(key == 'getUserEmail') job.setUserEmail(value);
-				if(key == 'getUserName') job.setUserName(value);
-				if(key == 'getUserFullName') job.setUserFullName(value);
-				if(key == 'getHierarchyPath') job.setHierarchyPath(splitString(value, ","));
-
-				if(key == 'getUniqueNamePrefix') oldUniqueNamePrefix = value;
-				if(key == 'getName') oldName = value;
-			}
-
-		} else {
-			// Private data
-			if(verboseDebugging === true){
-				s.log(-1, "Found private data");
-			}
-
-			pdChildren = node.getChildNodes();
-
-			// For each PD
-			for(index = 0; index < pdChildren.getCount(); index++){
-
-				pdNode = pdChildren.getItem(index);
-				key = pdNode.getAttributeValue('key');
-
-				if(verboseDebugging === true){
-					s.log(-1, "Restoring PD key: "+key);
-				}
-
-				textNode = pdNode.getFirstChild();
+		return false;
+//		fatalJobCleaner(s, job, filePath); // Kill the failure
+	} else {
+		doc = new Document(jobTicketPath);
+		if(!doc.isWellFormed()) {
+			// Something is wrong, read the doc for debugging
+			malformedTicket = File.read(jobTicketPath, "UTF-8");
+			job.log(3, "Malformed ticket: "+ malformedTicket);
+			// Fail the dummy job
+			job.fail("Job ticket could not be restored and the job could not be processed.");
```

Comment:
It might be confusing have two of the same log messages in different blocks. For debugging purposes, I'd change this one in some way so you distinguish between the two.